wbooks/src/app/screens/Library/LibraryWrapper/styles.ts

```diff
@@ -0,0 +1,9 @@
+import { StyleSheet } from 'react-native';
+import { colors } from '@constants/colors';
+
+export const styles = StyleSheet.create({
```

Comment:
2 things:
* `export default StyleSheet.create({`
* Move this file to `wbooks/src/app/screens/Library/styles.ts`

Reply:
Refactored